.changeset/friendly-planes-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"@lens-protocol/react-web": patch
+---
+
+adds use client directive to entry point
```

Comment:
Can we use `**fix:**`, `**chore:**`, `**feat:**` prefix so that changelog is easy to read and share in release notes.

Reply:
I usually use more or less the same copy of the commit (or PR title if contains more than one commit).

Reply:
sure